src/urh/signalprocessing/IQArray.py

```diff
@@ -241,3 +241,48 @@ def export_to_wav(self, filename, num_channels, sample_rate):
         f.setframerate(sample_rate)
         f.writeframes(self.convert_to(np.int16))
         f.close()
+
+    def export_to_sub(self, filename, frequency=433920000, preset="FuriHalSubGhzPresetOok650Async"):
+        arr = []
+        counter = 0
+
+        for value in self.convert_to(np.uint8):
+            # if origin was a sub-file value is uint8, else value is [uint8, uint8]
+            # TODO: export only works with sub files. Idea: Export from bit stream?
+            try:
+                value = np.linalg.norm(value)
+            except:
+                pass
```

Comment:
Can we catch a more specific Exception here or at least add a comment when `np.linalg.norm` fails?